api/src/main/java/com/exelaration/abstractmemery/services/FileStorageService.java

```diff
@@ -0,0 +1,13 @@
+package com.exelaration.abstractmemery.services;
+
+import com.exelaration.abstractmemery.domains.Image;
+import org.springframework.web.multipart.MultipartFile;
+
+public interface FileStorageService {
+
+  public static final String uploadingDir = "/app/src/main/resources/images/";
+
+  public Image save(MultipartFile file);
+
+  public void createDirectory();
```

Comment:
This probably shouldn't be in the interface.  It should probably be a private helper method in the implementation.